src/main/java/meditracker/logging/MediLoggerFormatter.java

```diff
@@ -0,0 +1,63 @@
+package meditracker.logging;
+
+import java.util.logging.Formatter;
+import java.util.logging.LogRecord;
+
+
+//@@author annoy-o-mus-reused
+// Reused from https://stackoverflow.com/a/53211725
```

Comment:
You may want to consider removing the additional space before the word "Reused".  
Following the website example, https://nus-cs2113-ay2324s2.github.io/website/admin/appendixB-policies.html#policy-on-reuse

Reply:
resolved: 58cd06a648bc7d3ccb74a09031a6a2176ce6b7bc